config.yml

```
@@ -25,6 +25,11 @@ Metrics/AbcSize:
 
 Metrics/MethodLength:
   Max: 15
+  CountAsOne: [hash, array, method_call]
+
+RSpec/ExampleLength:
+  Max: 15
```

@Joel132 qué te parece si dejamos esto, no sé, en `10`? Para no relajarlo tanto porque el valor por defecto es `5`.

La idea de este cop es que te obliga a darle un buen uso a los helpers como `let`, y `before`.

```suggestion
  Max: 10
```